### PR TITLE
Update vzbot_base.def.json

### PR DESCRIPTION
**Pull request by ckvsoft**
Adjusted some default values

### CODE REVIEW

**Comment by jellespijker** [comment]
Can you sync your fork, with our `main`. We have added a **printer-linter** tool for our printer definitions and profiles. This should help us with processing our PR's. It performs some standard diagnostic checks and apply auto-formatting on the changed files in this PR, but you can also run it locally. See https://github.com/Ultimaker/Cura/tree/main/printer-linter

If it finds issues for which it knows a fix it should post a comment with that suggested fix such that you can easily commit that fix if you're okay with it.

**Comment by ckvsoft** [comment]
I fix that manual.

python3 printer-linter/src/terminal.py "vzbot_330.def.json" "vzbot_base.def.json" --fix --format
gives me
Diagnostics: []

File: resources/definitions/vzbot_base.def.json [hunk 1]
```diff
@@ -50,13 +50,14 @@
...
-        "retraction_hop_enabled":                       { "default_value": true },
+        "retraction_speed":                             { "default_value": 45, "maximum_value_warning": 150 },
+        "retraction_retract_speed":                     { "maximum_value_warning": 150 },
+        "retraction_prime_speed":                       { "value": "math.ceil(retraction_speed * 0.4)", "maximum_value_warning": 150 },
         "retraction_hop":                               { "default_value": 0.2 },
         "retraction_combing":                           { "value": "'noskin'" },
+        "z_seam_type":                                  { "value": "'back'" },
         "retraction_combing_max_distance":              { "default_value": 10 },
+        "retraction_hop_enabled":                       { "default_value": false },
         "travel_avoid_other_parts":                     { "default_value": false },
```
[the remark this answers is not in the document]
Comment (ckvsoft) [reply resources/definitions/vzbot_base.def.json:61]: its removed now,

**Review by Joeydelarago — APPROVED** [review]

**Comment by Joeydelarago** [comment]
Looks good! Thanks for the contribution 👍

**Pull request closed without merging**